pytorch_blade/torch_blade/onnx_backends/backend_conversion.py

```diff
@@ -37,7 +39,20 @@ def _deduplicate_onnx_graph_outputs(graph):
         visited.add(v)
 
 
-def _build_onnx_engine(subgraph, engine_build_func, q_info=None, dynamic_settings=None):
+def _try_cast_graph_integer_inputs_to_i32(graph):
+    # Refer to https://github.com/onnx/onnx-tensorrt/blob/main/docs/operators.md.
+    # There is limited support for INT32, INT64, and DOUBLE types.
+    # Convert INT64 to INT32.
+    for val in graph.inputs():
+        if val.type().scalarType() == "Long":
```

Comment:
Maybe we should also handle double -> float conversion? (or at least add a todo?)